### PR TITLE
Update UI of verification view

### PR DESCRIPTION
**Pull request by vahinim**
Update UI of business_verification_view.dart. Corresponding image is attached below.

verification screen

### CODE REVIEW

**Comment by vahinim** [comment]
I have updated UI of main verification page. The first two commits are merged by mistake and I wasn't able to delete those specific commits. So I finally deleted unused files.

File: frontend/gpay_agent/lib/business_verification_views/business_registration_details_view.dart [hunk 1]
```diff
@@ -61,7 +61,7 @@ class _MerchantHomeViewState extends State<MerchantHomeView> {
       floatingActionButton: FloatingActionButton.extended(
         onPressed: () {
           Navigator.push(context,
-              MaterialPageRoute(builder: (context) => VerificationHomeView()));
+              MaterialPageRoute(builder: (context) => MyHomePage()));
```
Comment (trblessy) [inline frontend/gpay_agent/lib/business_verification_views/business_registration_details_view.dart]: Please give a more descriptive name.

File: frontend/gpay_agent/lib/business_verification_views/business_verification_view.dart [hunk 2]
```diff
@@ -1,173 +1,472 @@
...
+    );
+  }
+
+  /// Defines action to be taken when pop up menu items are selected.
   void choiceAction(String choice) {
     if (choice == VerificationMenuItems.Cancel) {
-      print('Discard Verification');
       //When ever user clicks on discard verification it opens up verification failed
       //TODO: Need to return to home page after cancelling verification
+      //runApp(MyApp());
```
Comment (trblessy) [inline frontend/gpay_agent/lib/business_verification_views/business_verification_view.dart]: please remove comments that are not required.

File: frontend/gpay_agent/lib/business_verification_views/business_verification_view.dart [hunk 3]
```diff
@@ -1,173 +1,472 @@
...
+        return 'Merchant Name';
+      case headerName.business_details:
+        return 'Address';
+      case headerName.business_phone:
+        return 'Phone Number';
     }
   }
 
-  /// Displays dialog box confirms whether FOS Agent want to verify or not.
+  /// returns the value of the field fetched from server to be displayed based on header name.
```
Comment (trblessy) [inline frontend/gpay_agent/lib/business_verification_views/business_verification_view.dart]: Returns

**Review by trblessy — APPROVED** [review]

File: frontend/gpay_agent/lib/business_verification_views/business_verification_view.dart (hunk 2 above)
Comment (vahinim) [reply frontend/gpay_agent/lib/business_verification_views/business_verification_view.dart]: removed

File: frontend/gpay_agent/lib/business_verification_views/business_verification_view.dart (hunk 3 above)
Comment (vahinim) [reply frontend/gpay_agent/lib/business_verification_views/business_verification_view.dart]: changed

File: frontend/gpay_agent/lib/business_verification_views/business_registration_details_view.dart (hunk 1 above)
Comment (vahinim) [reply frontend/gpay_agent/lib/business_verification_views/business_registration_details_view.dart]: changed

**Pull request merged**